gtsfm/frontend/matcher/dummy_matcher.py

```diff
@@ -37,7 +37,7 @@ def match(
 
         # check if we have non-zero descriptors in the both image
         if descriptors_im1.size == 0 or descriptors_im2.size == 0:
-            return np.array([], dtype=np.int)
+            return np.array([], dtype=np.uint8)
```

Comment:
Thanks for the PR, Ayush. Is int32 better? could have more than 256 of them?

Reply:
Yes. My bad. Will make this change.